build-system/server/new-server/transforms/utilities/cdn.ts

```diff
@@ -1,8 +1,7 @@
-
-
 import {URL} from 'url';
-import {parse, format, basename} from 'path';
+import {basename, format, parse} from 'path';
 
+// eslint-disable-next-line local/no-forbidden-terms
```

Comment:
We ban direct usage of `"cdn.ampproject.org"` in code, since we want the CDN to be configurable. 
We should probably get this from a config file instead (src/config.js), but I leave that for a different PR.

cc @erwinmombay